resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -1,6 +1,8 @@
 Try {
     $conemulocation = "$env:programfiles\ConEmu\Conemu64.exe"
-    $chefdkinit = 'chef shell-init powershell | iex; Import-Module chef; $env:CHEFDK_ENV_FIX = 1'
+    # We don't want the current path to affect which "chef shell-init powershell" we run, so we need to set the PATH to include the current omnibus.
+    $chefdk_bin = (split-path $MyInvocation.MyCommand.Definition -Parent)
+    $chefdkinit = '"$env:PATH = ''' + $chefdk_bin + ';'' + $env:PATH; $env:CHEFDK_ENV_FIX = 1; chef shell-init powershell | out-string | iex; $OldWarningPreference = $WarningPreference; $WarningPreference = ''SilentlyContinue''; Import-Module chef; $WarningPreference = $OldWarningPreference"'
```

Comment:
Are you suppressing warnings to avoid warnings about the cmdlet verbs? You could call Import-Module with -DisableNameChecking to do that.